python/S3_BUCKET_ENCRYPTED_KMS/S3_BUCKET_ENCRYPTED_KMS.py

```diff
@@ -0,0 +1,83 @@
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+S3_BUCKET_ENCRYPTED_KMS
+
+Description:
+Check whether Amazon S3 bucket is encrypted with KMS key.
+
+Rationale:
+Ensure that the Amazon S3 bucket is always encrypted with the KMS key so that no clear text documents.
+
+Indicative Severity:
+High
+
+Trigger:
+Periodic
```

Comment:
I am confused, S3 Bucket CI seem to record the encryption information in "ServerSideEncryptionConfiguration". Why not Configuration Change?